### ISSUE TITLE
`hey` command fails with "syntax error near unexpected token <"

### ISSUE THREAD

**Issue by ryanj** [openshift-labs/learn-katacoda#995]
In section 5 of Getting Started with OpenShift Serverless:

Example command `hey -n 2550 -c 850 -t 60 "http://prime-generator-serverless-tutorial.2886795292-80-host18nc.environments.katacoda.com/?sleep=3&upto=10000&memload=100" && oc get deployment -n serverless-tutorial` fails with:

```
/usr/local/bin/hey: line 1: syntax error near unexpected token `<'
/usr/local/bin/hey: line 1: `<?xml version='1.0' encoding='UTF-8'?><Error><Code>AccessDenied</Code><Message>Access denied.</Message><Details>Anonymous caller does not have storage.objects.get access to the Google Cloud Storage object.</Details></Error>'
```

**Comment by ryanj** [comment]
Workaround: 
1. Add `yum install -y  httpd-tools` to `init-env.sh` 
2. Use `ab` instead of `hey` (same flags):
```
ab -n 2550 -c 850 -t 60 "http://prime-generator-serverless-tutorial.2886795292-80-host18nc.environments.katacoda.com/?sleep=3&upto=10000&memload=100" && oc get deployment -n serverless-tutorial```

**Comment by ryanj** [comment]
resolved by #1119
